array.js

```diff
@@ -0,0 +1,14 @@
+function analyze(array) {
+  const average = array.reduce((prev, cur) => prev + cur, 0) / array.length;
+  const min = array.reduce((prev, cur) => (prev < cur ? prev : cur));
+  const max = array.reduce((prev, cur) => (prev > cur ? prev : cur));
+  const { length } = array;
+
+  return {
+    average,
+    min,
+    max,
+    length,
+  };
+}
```

Comment:
You can also turn the whole function to an arrow function if you are into that kind of things, for example:

```js
const analyze = array => ({
  average: array.reduce((prev, cur) => prev + cur, 0) / array.length,
  min: array.reduce((prev, cur) => (prev < cur ? prev : cur)),
  max:  array.reduce((prev, cur) => (prev > cur ? prev : cur)),
  length: array.length,
})
```